examples/wikipedia.ts

```diff
@@ -40,10 +40,11 @@ async function main() {
     );
   }
   const logger = new Logger(["info"], (msg) => console.log(msg));
+  const agent = new Agent({ modelApi: chatApi });
 
   const agentBrowser = new AgentBrowser({
-    agent: new Agent({ modelApi: chatApi }),
-    browser: await Browser.create(argv.headless),
+    agent: agent,
```

Comment:
as a cute small thing, you can just do

```
agent,
```

as shorthand when the value is a var with the same name as the key